src/config/dynamic-theme-fixes.config

```diff
@@ -6031,6 +6031,7 @@ div[data-label="nd"] > div > div > svg > path[fill="#000000"]
 div[role="document"] > div[role="button"] .a-b-c
 div > svg > circle[fill="white"]
 img[src*="empty_state_trash"]
+div[role="document"]
```

Comment:
It feels bit too general to include this. Should we include `div[role="dialog"]`?

![image](https://user-images.githubusercontent.com/25481501/195158912-3af0c81a-ed0d-4df0-a482-688322478d82.png)

Reply:
I did not see any problems with my rule, but just in case, I clarified the rule on your recommendation.

Reply:
I've never had a great experience with using "general"-looking selectors.